src/App.vue

```diff
@@ -107,33 +129,166 @@ const resetComparison = () => {
   comparisonResult.value = null;
   error.value = null;
 };
+
+const downloadCsv = async () => {
```

Comment:
this can probably be extracted into a helper function file or a composable or something